### PR TITLE
solves 8

### PR DESCRIPTION
**Pull request by jchancoding**

### CODE REVIEW

**Comment by jchancoding** [comment]
Solves 8 and passes tests

File: curriculum/js1/8.js [hunk 1]
```diff
@@ -5,8 +5,21 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, counter = 1, GCD = 1) => {
```
Comment (songz) [inline curriculum/js1/8.js]: never name a variable all caps. Unless its a global constant, never capitalize first character if the variable name.

File: curriculum/js1/8.js [hunk 2]
```diff
@@ -5,8 +5,21 @@
...
+  //stops once counter reaches the lesser of a and b
+  if (b >= a && counter > a) {
```
Comment (songz) [inline curriculum/js1/8.js]: You only need 1 if check: `if(counter > a || counter > b)`

**Comment by jchancoding** [comment]
Fixes a/b + counter base case comparison and lowercases GCD variable

**Review by songz — APPROVED** [review]

**Pull request merged**